mlir/test/lib/Dialect/MemRef/TestEmulateNarrowType.cpp

```diff
@@ -118,6 +119,11 @@ struct TestEmulateNarrowTypePass
       *this, "skip-memref-type-conversion",
       llvm::cl::desc("disable memref type conversion (to test failures)"),
       llvm::cl::init(false)};
+
+  Option<bool> atomicStore{
+      *this, "atomic-store",
+      llvm::cl::desc("use atomic store instead of load-modify-write"),
+      llvm::cl::init(true)};
```

Comment:
`atomic-store` doesn't reveal what the default is (enable vs disable). Could you rename this as `disable-atomic-store` (or `disable-atomic-rmw`) and then set the default to `false`? The overall effect will be the same (use atomic RMW by default), but at least the flag name will be more descriptive.